.github/workflows/nightly-build.yml

```
@@ -0,0 +1,41 @@
+name: Nightly build
+
+on:
+  schedule:
+    # only on weekdays
+    - cron:  '0 0 * * 2-6'
+  workflow_dispatch:
```

I think, because there's no further configuration defined for this event, you could also write:

```suggestion
  workflow_dispatch
```

What do you think?

---

@jayjayjpg I believe even when being empty you still need the `:`.